sui_types/src/crypto.rs

```diff
@@ -301,6 +301,8 @@ impl Signature {
 
 pub struct AuthoritySignature(pub dalek::Signature);
 
+impl BcsSignable for AuthoritySignature {}
```

Comment:
Where do we need this?

Reply:
The check function called on the `signed_batch` requires that the value implements `BcsSignable`.

Reply:
Fixed.